src/main/java/decodes/tsdb/CpCompDependsUpdater.java

```diff
@@ -858,15 +864,19 @@ private boolean compModified(DbKey compId)
 		{
 			// Have to remove comp-depends for the now-disabled or deleted comp.
 			String q = "DELETE FROM CP_COMP_DEPENDS WHERE COMPUTATION_ID = " + compId;
```

Comment:
Needs to be prepared statement with bind variable. This is call so often that it eats the shared query pool.